src/makeCalculator.js

```diff
@@ -4,7 +4,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(number) {
+      this.result += number;
+    },
+
+    subtract(num) {
+      this.result -= num;
+    },
+
+    divide(num) {
+      this.result /= num;
+    },
```

Comment:
The `divide` method correctly uses the `/=` operator, following the [CODE STYLE] checklist.